docs/memcached.md

```diff
@@ -57,7 +29,3 @@ These are the metrics available for this scraper.
 | operation | The type of operation. | <ul> <li>increment</li> <li>decrement</li> <li>get</li> </ul> |
 | state | The type of CPU usage. | <ul> <li>system</li> <li>user</li> </ul> |
 | type | Result of cache request. | <ul> <li>hit</li> <li>miss</li> </ul> |
```

Comment:
Same here. Can we remove the HTMl tags from the markdown?

| Name | Description | Values |
| ---- | ----------- | ------ |
| command | The type of command. | get, set, flush, touch
| direction | Direction of data flow. | sent, received
| operation | The type of operation. | increment, decrement, get |
| state | The type of CPU usage. | system, user |
| type | Result of cache request. | hit, miss |

Reply:
Done